tokio/src/net/tcp/socket.rs

```diff
@@ -86,6 +87,11 @@ cfg_net! {
     }
 }
 
+cfg_net! {
+    #[allow(unreachable_pub)] // https://github.com/rust-lang/rust/issues/57411
+    pub use mio::net::TcpKeepalive;
```

Comment:
We are not exposing mio as part of Tokio's public API. If we do this, we will need to re-define the type.